controllers/RssController.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+
+namespace app\controllers;
+
+
+use app\models\Extension;
+use app\models\News;
+use app\models\Wiki;
+use DateTime;
+use yii\base\Exception;
+use yii\helpers\Url;
+use yii\web\Controller;
+use Zend\Feed\Writer\Feed;
+
+class RssController extends Controller
+{
+    public function actionAll()
+    {
+        $feed = new Feed();
+        // Title of the feed
+        $feed->setTitle('Live News for Yii Framework');
+        // Link to the feed's target, usually a homepage:
+        $feed->setLink('http://www.yiiframework.com/');
+        // Link to the feed itself, and the feed type:
+        $feed->setFeedLink('http://www.yiiframework.com/rss.xml', 'rss');
+
+        // Feed description; only required for RSS:
+        $feed->setDescription('News, fresh extensions and wiki articles about Yii framework.');
+
+        /** @var News[]|Extension[]|Wiki[] $data */
+        $data = array_merge(
+            $this->getNews(),
+            $this->getExtensions(),
+            $this->getWikiPages()
+        );
+
+        asort($data);
```

Comment:
you index data by `created_at` so this should be `krsort()` instead to sort by date DESC.

also wouldn't the `array_merge` above overwrite items that have been created on the same date?

better use ArrayHelper::multisort() on 'created_at' and do not index items.